crate/crash/command.py

```diff
@@ -139,6 +139,10 @@ def _conf_or_default(key, value):
     parser.add_argument('--hosts', type=str, nargs='*',
                         default=_conf_or_default('hosts', ['localhost:4200']),
                         help='connect to HOSTS.', metavar='HOSTS')
+    parser.add_argument('--timeout', type=str, metavar='TIMEOUT',
+                        help='Configure "connect,read" network timeout values different than "5,infinite" seconds.'
```

Comment:
just double checking, `infinite` is a valid value that is parsed by `urllib3.Timout()`, right?

Reply:
No, `"infinite"` isn't a valid value, it is just a surrogate. When obtaining Python's `None` symbol, the timeout will be infinite. Currently, there is no way to obtain this value from a text representation from the command line.

Reply:
Do you have a suggestion how to improve code or docs around this topic?

Reply:
Yeah. In this case, I think it would be more about discussing the need to find a textual representation. Would you prefer that the literal `"None"` would be translated to `None`, or would you prefer that `inf` or `infinite` would be translated to `None`?

Without implementing that, I think there would currently be no possibility to configure, for example, an infinite connect timeout. We would need to check / validate what `--timeout=,42` does, and eventually shape it a bit more to converge this merely undefined behaviour into something with a more sensible semantic.

Reply:
Currently, omitting one or both values fails like:
```
$ crash --timeout=42,
$ crash --timeout=,42
$ crash --timeout=,
could not convert string to float: ''
```

So, momentarily, only those invokations work, adjusting either the connect timeout value, or both.
```
$ crash --timeout=42
$ crash --timeout=42,42
```

Maybe it is not needed to improve this further? Maybe it would be needed to improve error handling and presentation to the user? WDYT?

Reply:
imho, we should give the chance to the user to configure the default "infinite" value, if needed,
so I'd go with `None` (case insensitive though, so that none, NONE, NoNE, etc. is all the same)

Reply:
Good call. 0 or -1 are commonly used to set no-timeout/infinite. +1

Reply:
The patch has been amended to implement your suggestion. Now, `-1` can be used as a surrogate value to represent `None`, which means "infinite timeout".

Reply:
The patch has been significantly amended, so I am humbly asking for another review. Thank you again.